.github/workflows/ci-ubuntu.yml

```diff
@@ -32,82 +32,58 @@ on:
 ##
 ########################################################################
 
+########################################################################
+## SETTINGS
+##
+## AGDA_BRANCH picks the branch of Agda to use to build the library.
+########################################################################
+
 env:
-  GHC_VERSION: 8.6.5
-  CABAL_VERSION: 3.2.0.0
+  AGDA_BRANCH: v2.6.3
```

Comment:
Suggestion: Add a comment that movable identifiers like `master` would not work here because they do not make adequate cache keys.
I.e., if I put `master` here, this CI would restore the Agda it last built from the cache rather than building Agda anew when `master` changes.